src/Uno.Foundation/FoundationFeatureConfiguration.cs

```diff
@@ -12,9 +12,45 @@ public static class FoundationFeatureConfiguration
 		/// </summary>
 		internal static void RestoreDefaults()
 		{
+			GestureRecognizer.RestoreDefaults();
 			Rect.RestoreDefaults();
 		}
 
+		public static class GestureRecognizer
```

Comment:
That's probably placed too low. Why not in Uno.UWP ?

Reply:
Right it's one layer too low, going to fix it.